### ISSUE TITLE
Cursor height not set initially

### ISSUE THREAD

**Issue by D3LT4PL** [voxeltycoon/bug-tracker#197]
**Describe the bug**
It's a very minor bug, the cursor height is set to 0 (instead of the actual value) initially after selecting the bridge build mode. It changes to the right value after changing the height.

**Steps to reproduce**

1. Try to build road with bridge mode
2. Cursor height is 0

**Build version**
0.85.0.24 steam

**Platform**
Windows 10  (10.0.19042) 64bit

**Reproducibility**
Every time

**Logs/saves**
Any save or new world

**Comment by polak-jan** [comment]
I can confirm this. Though from my limited testing it doesn't seem like the cursor height isn't set, rather that the UI isn't updated and the value that was there when either the tunnel or bridge tools were used.

**Comment by D3LT4PL** [comment]
Yes, the value is set when building, but the value in the UI is not set
